src/transform.ts

```diff
@@ -226,7 +242,7 @@ export function transformClassPair(
           last: i === params.length - 1,
         })),
         returnType: typeToString(method.properties.returns).replace(/^null$/, 'void'),
-        throws: method.properties.throws ? typeToString(method.properties.throws).split(' | ') : [],
+        throws: findRefs(method.properties.throws).map(deref),
```

Comment:
I don't understand `findRefs` here.  E.g. if it throws
```ts
class DidntThrowFooError {
  // ...
  public notThrown: FooError;
}
```
then AFAICT this says it throws a `FooError`.

Reply:
Cool, the comment explains it!